src/main/java/nl/tudelft/pixelperfect/event/EventFactory.java

```diff
@@ -0,0 +1,61 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Created by woute on 5/10/2016.
+ */
+
+/**
+ * Not a factory as the design pattern, but used for id's.
+ * <p>
+ * Create a random event, with unique id's.
+ */
+public class EventFactory {
+
+  private int id;
+
+  /**
+   * Constructor for EventFactory.
+   */
+  public EventFactory() {
+    id = 0;
+  }
+
+  /**
+   * Generate a random event.
+   *
+   * @return an Event.
+   */
+  public Event randomEvent() {
+    int x = (int) Math.floor(Math.random() * 100) % 4;
```

Comment:
I recommend using the Random class rather than the Math class for generating random numbers.
http://stackoverflow.com/questions/738629/math-random-versus-random-nextintint